src/any_agent/frameworks/any_agent.py

```diff
@@ -159,26 +166,53 @@ def run(self, prompt: str, **kwargs: Any) -> AgentTrace:
             self.run_async(prompt, **kwargs)
         )
 
-    def serve(self, serving_config: ServingConfig | None = None) -> None:
+    def serve(
+        self, serving_config: ServingConfig | None = None, log_level: str = "warning"
```

Comment:
Does it make sense for the `log_level` be a part of the ServingConfig, instead of a new parameter?

Reply:
Yep 👍🏼 Didn't think about that one.